sockets.js

```diff
@@ -502,6 +549,9 @@ if (cluster.isMaster) {
 			process.send(`!${socketid}`);
 			sockets.delete(socketid);
 			channels.forEach(channel => channel.delete(socketid));
+			socketid = null;
+			socketip = null;
+			socket = null;
```

Comment:
Do you really need to clean temporary variables when they contain primitives? Even if they were pointers I'd consider this excessive. This should make zero difference...

Reply:
You're probably right. I was being overzealous here